salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/AccountHooks.java

```diff
@@ -37,7 +51,52 @@ public void deleteAccount() {
     }
 
     /**
-     * [SL] Hook that delete an Account saved in Context via API.
+     * [MR] Hook that create many account from a Csv file through Salesforce API.
+     *
+     * @throws IOException
+     */
+    @Before(value = "@createAccountsFromCsv")
+    public void createAccountsFromCsv() throws IOException {
+        try {
+            RequestManager.get("/Account/");
+        } catch (IllegalArgumentException e) {
+            String sourceFile = "src/test/resources/files/csv/preconditionAccounts.csv";
+            RequestManager.setRequestSpec(ApiAuthenticator.getLoggedReqSpec(
+                    context.getUserByAlias("Account Owner User")));
+            List<Map<String, String>> accountsList = CSVReader.getListOfMapsFromCsvFile(sourceFile);
+            for (Map<String, String> accountInfo : accountsList) {
+                String body = "{\n";
+                for (String key : accountInfo.keySet()) {
+                    body += "\"" + key + "\": \"" + accountInfo.get(key) + "\", \n";
+                }
+                body = body.substring(0, body.length() - CHARS_TO_TRIM) + "\n}";
```

Comment:
This section of code need inside the CSVReader but before is need change the name of class to a class more descriptive maybe CSVUtils and inside this class this function mm maybe CSVToJson